pylinalg/func/misc.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+
+
+def aabb_to_sphere(aabb, /, *, out=None, dtype=None):
+    """A sphere that envelops an Axis-Aligned Bounding Box.
+
+    Parameters
+    ----------
+    aabb : ndarray, [2, 3]
+        The axis-aligned bounding box.
+    out : ndarray, optional
+        A location into which the result is stored. If provided, it
+        must have a shape that the inputs broadcast to. If not provided or
+        None, a freshly-allocated array is returned. A tuple must have
+        length equal to the number of outputs.
+    dtype : data-type, optional
+        Overrides the data type of the result.
+
+    Returns
+    -------
+    sphere : ndarray, [4]
+        A sphere (x, y, z, radius).
+
+    """
+
+    raise NotImplementedError()
+
+
+def aabb_transform(aabb, homogenious_matrix, /, *, out=None, dtype=None):
```

Comment:
well, first of all, I think there's a spelling mistake here in the parameter `homogenious_matrix`, but also, I've called it `transform` or `matrix` elsewhere, I think we should be consistent where we can. my vote goes to `transform`, because I don't really know what `homogeneous` means in relation to a matrix